core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/redirect/RedirectViewModel.kt

```diff
@@ -87,7 +87,7 @@ class RedirectViewModel(
                     put(
                         ARG_LEARNINGSPACE_URL,
                         apiUrlConfig.presetLearningSpaceUrl?.requireHttpPrefix()
-                            ?.requirePostfix("/")?.replace("/umapp", "") ?: ""
+                            ?.requirePostfix("/")?: ""
```

Comment:
The /umapp shouldn't have been there - it's not part of the learning space url ever. 

For style:

if you used:
```
val presetLearningSpaceUrl = apiUrlConfig.presetLearningSpaceUrl
```

Then you could remove null safety within the if statement. That should be done